src/Driver.cpp

```diff
@@ -81,11 +81,11 @@ int Driver::checkParticularResponse(string const& buffer) const
 {
     string::size_type eol = buffer.find("\r\n\r\n");
     if(eol != string::npos)
-	// Add \r\n\r\n to buffer.
-	return eol+4;
+        // Add \r\n\r\n to buffer.
+        return eol+4;
     // Max observed Particular Response: AT&V (get parameters) with 345 in length
     else if(buffer.size() > 400)
-	return -1;
+        throw runtime_error("CheckParticularResponse: Received a too big buffer of size \"" +to_string(buffer.size())+ "\". Check buffer \"" +usblParser.printBuffer(buffer)+ "\"");
```

Comment:
Yep, that is what I had in mind. Still, I do think it would be best to revert this IF we manage to figure it out what really is wrong with the whole usbl comms